backend/db.py

```diff
@@ -72,8 +72,8 @@ def get_census_division_aggregate():
         CREATE EXTENSION IF NOT EXISTS tablefunc;
         SELECT *
         FROM census_division
-        JOIN ({}) sp ON census_division.id = sp.cd_id
-        JOIN ({}) demographics ON census_division.id = demographics.census_division_id
+        LEFT JOIN ({}) sp ON census_division.id = sp.cd_id
+        LEFT JOIN ({}) demographics ON census_division.id = demographics.census_division_id
```

Comment:
We would like census divisions to be returned even if there's no service provider data so LEFT JOIN is the way to go.